.travis.yml

```diff
@@ -1,15 +1,27 @@
+sudo: required
+
+dist: trusty
+
 language: python
+
 python:
   - 2.7
   - 3.5
 
 before_install:
   - sudo apt-get install -y python-setuptools
+  - strings /usr/lib/x86_64-linux-gnu/libstdc++.so.6 | grep GLIBCXX
```

Comment:
i don't think this is needed, we can just take it out. same for the `pip freeze` command